README.md

```diff
@@ -6,9 +6,9 @@ Project template for django based projects, optimized for making REST API with d
 ## Features
 
 - Django 2.0.x
-- Python 3.6.x 
+- Python 3.6.x
 - [12-Factor][12factor] based settings management via [django-environ], reads settings from `.env` if present.
-- PostresSQL everywhere (support of postgis is available)
+- Supports PostresSQL 10.0 (support of postgis-2.4 is available).
```

Comment:
Minor typo: it should be `PostgreSQL`, rest is good to merge! :)

Reply:
Also, i think it's worth adding the same to generated README inside cookiecutter folder as well, thoughts?

Reply:
Sounds good to me!

Updated.